core/templates/dev/head/pages/splash-page/splash-page.controller.ts

```diff
@@ -22,56 +22,61 @@ require('domain/utilities/url-interpolation.service.ts');
 require('services/site-analytics.service.ts');
 require('services/user.service.ts');
 
-angular.module('oppia').directive('splashPage', [function() {
-  return {
-    restrict: 'E',
-    scope: {},
-    bindToController: {},
-    template: require('!html-loader!./splash-page.directive.html'),
-    controllerAs: '$ctrl',
-    controller: [
-      '$rootScope', '$timeout', '$window', 'SiteAnalyticsService',
-      'UrlInterpolationService', 'UserService',
-      function($rootScope, $timeout, $window, SiteAnalyticsService,
-          UrlInterpolationService, UserService) {
-        var ctrl = this;
-        ctrl.userIsLoggedIn = null;
-        $rootScope.loadingMessage = 'Loading';
-        UserService.getUserInfoAsync().then(function(userInfo) {
-          ctrl.userIsLoggedIn = userInfo.isLoggedIn();
-          $rootScope.loadingMessage = '';
-        });
-        ctrl.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
-        ctrl.getStaticSubjectImageUrl = function(subjectName) {
-          return UrlInterpolationService.getStaticImageUrl('/subjects/' +
-            subjectName + '.svg');
-        };
+angular.module('oppia').directive('splashPage', [
+  'UrlInterpolationService', function(
+      UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      bindToController: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/splash-page/splash-page.directive.html'),
```

Comment:
We use require for templates in some directives now, please do not change it back to `templateUrl`.